tools/docker/run.sh

```diff
@@ -6,11 +6,10 @@
 # See LICENSE file for more details.
 ###############################################################
 
-scriptdir=$(dirname "$(readlink -f "$0")")
-rootdir=$(realpath "$scriptdir/../..")
+# shellcheck source=../../tools/functions.sh
```

Comment:
We're going in circles with this shellcheck source stuff...

Let's wait for #1204 to be merged, and not change it anymore after that.

Reply:
OK.